src/config.ts

```diff
@@ -0,0 +1,5 @@
+const config = {
+  DEFAULT_EXPIRATION_HOURS: 7 * 24,
+  DYNAMODB_TABLE: "stackJanitorTable"
```

Comment:
These should be environment variables, with the defaults set once in the severless.yml

Reply:
Nice idea! Done.